### PR TITLE
Toolkit Identify:

### PR DESCRIPTION
**Pull request by lsmallwood**
@michael-tims please review the addition of the signals to the toolkit

### CODE REVIEW

File: Plugin/CppApi/include/ToolResourceProvider.h [hunk 1]
```diff
@@ -74,14 +75,20 @@ public slots:
   void onMousePressedAndHeld(QMouseEvent& mouseEvent);
   void onMouseDoubleClicked(QMouseEvent& mouseEvent);
   void onIdentifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
+  void onIdentifyGraphicsOverlaysCompleted(QUuid id, QList<IdentifyGraphicsOverlayResult*> identifyResults);
+  void onIdentifyLayerCompleted(QUuid taskId, IdentifyLayerResult* identifyResult);
```
Comment (ldanzinger) [inline Plugin/CppApi/include/ToolResourceProvider.h]: should we keep the parameter consistently `id` throughout?

File: Plugin/CppApi/include/ToolResourceProvider.h [hunk 2]
```diff
@@ -74,14 +75,20 @@ public slots:
...
+  void onIdentifyLayersCompleted(QUuid taskId, QList<IdentifyLayerResult*> identifyResults);
   void onScreenToLocationCompleted(QUuid taskId, const Point& location);
 
 signals:
   void sceneChanged();
   void geoViewChanged();
   void mapChanged();
   void spatialReferenceChanged();
-  void identifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
```
Comment (ldanzinger) [inline Plugin/CppApi/include/ToolResourceProvider.h]: i don't think we want to change the signature of the signal, do we? Maybe leave this one and use `id` for all the others?

File: Plugin/CppApi/include/ToolResourceProvider.h (hunk 1 above)
Comment (lsmallwood) [reply Plugin/CppApi/include/ToolResourceProvider.h]: i think taskId is more descriptive

**Comment by ldanzinger** [comment]
taskId sounds good. merging

**Pull request merged**